src/datastore/src/main/java/org/locationtech/geogig/geotools/data/GeogigFeatureStore.java

```diff
@@ -85,7 +85,7 @@ public void setTransaction(Transaction transaction) {
                 GeogigFeatureStore.this.setTransaction(transaction);
             }
         };
-        super.hints = delegate.getSupportedHints();
+        //super.hints = delegate.getSupportedHints();
```

Comment:
I saw this and wasn't sure what the implications were.  I think we do have some special hints in the FeatureWriter, but I'm not sure if this is necessary or not...